pymodbus/client/mixin.py

```diff
@@ -523,7 +523,7 @@ class DATATYPE(Enum):
     @classmethod
     def convert_from_registers(
         cls, registers: List[int], data_type: DATATYPE
-    ) -> Union[int, float, str]:
+    ) -> Union[float, str]:
```

Comment:
But it also returns an int ! if you do not specify that, the user might believe he needs to write 
```
   int(result)
```
to get rid of the decimals.